src/pretix/control/forms/mapping.py

```diff
@@ -0,0 +1,108 @@
+#
+# This file is part of pretix (Community Edition).
+#
+# Copyright (C) 2014-2020 Raphael Michel and contributors
+# Copyright (C) 2020-2021 rami.io GmbH and contributors
+#
+# This program is free software: you can redistribute it and/or modify it under the terms of the GNU Affero General
+# Public License as published by the Free Software Foundation in version 3 of the License.
+#
+# ADDITIONAL TERMS APPLY: Pursuant to Section 7 of the GNU Affero General Public License, additional terms are
+# applicable granting you additional permissions and placing additional restrictions on your usage of this software.
+# Please refer to the pretix LICENSE file to obtain the full terms applicable to this work. If you did not receive
+# this file, see <https://pretix.eu/about/en/license>.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied
+# warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Affero General Public License for more
+# details.
+#
+# You should have received a copy of the GNU Affero General Public License along with this program.  If not, see
+# <https://www.gnu.org/licenses/>.
+#
+import json
+
+from django import forms
+from django.forms import formset_factory
+from django.utils.translation import gettext_lazy as _
+
+from pretix.base.datasync.datasync import (
+    MODE_APPEND_LIST, MODE_OVERWRITE, MODE_SET_IF_EMPTY, MODE_SET_IF_NEW,
+)
+from pretix.base.datasync.sourcefields import QUESTION_TYPE_IDENTIFIERS
+
+
+class PropertyMappingForm(forms.Form):
+    pretix_field = forms.CharField()
+    external_field = forms.CharField()
+    value_map = forms.CharField(required=False)
+    overwrite = forms.ChoiceField(
+        choices=[
+            (MODE_OVERWRITE, _("Overwrite")),
+            (MODE_SET_IF_NEW, _("Fill if new contact")),
```

Comment:
contact is no longer correct wording outside of pretix-hubspot

Reply:
```suggestion
            (MODE_SET_IF_NEW, _("Fill if new")),
```